app/models/saved_claim/disability_compensation.rb

```diff
@@ -24,8 +24,8 @@ def self.from_hash(hash)
   def to_submission_data(user)
     form4142 = EVSS::DisabilityCompensationForm::Form4142.new(user, @form_hash.deep_dup).translate
     form526 = @form_hash.deep_dup
-    form526_uploads = form526['form526'].delete('attachments')
     dis_form = EVSS::DisabilityCompensationForm::DataTranslationAllClaim.new(user, form526, form4142.present?).translate
+    form526_uploads = form526['form526'].delete('attachments')
```

Comment:
I'm curious what the purpose of reordering was here. Should attachment data be included in the transformed version of the submission and it wasn't?

Reply:
Oops this was covered in a commit message before I split the PRs -

So this is really annoying, the way this was built the attachments array was being deleted from the form metadata while at the same time being assigned to a variable, `form526_uploads`, and then put into the translated metadata on line 32 below with a different key.

So if I wanted to handle the form overflow text in the same place as with the other notes (in `EVSS::DisabilityCompensationForm::DataTranslationAllClaim.`) I had to move the line that removes the attachments below, that way I still had access to the `attachments` array in `EVSS::DisabilityCompensationForm::DataTranslationAllClaim`. Does that make sense?

Reply:
Perfect sense. Thanks for the explanation.